Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -122,6 +122,8 @@ public override void ViewDidLoad()
 			ItemsSource = CreateItemsViewSource();
 			ItemsViewLayout.GetPrototype = GetPrototype;
 
+			CollectionView = CreateUICollectionView();
```

Comment:
What's this about?

Reply:
I need to handle some methods on UICollectionView to implment the looping